adapters/Survey/index.test.js

```diff
@@ -0,0 +1,42 @@
+import baseAdapter from 'adapters/Base';
+import surveyAdapter from '.';
+
+describe('surveyAdapter', () => {
+  describe('fetchSurveys', () => {
+    describe('given a current page and an authorization header', () => {
```

Comment:
Maybe adding this to make it a bit more clear 💡 

```suggestion
    describe('given a current page number and an authorization header', () => {
```

Reply:
done in ab05c62c262d7d61b22889922d4d4962fe36db17!